gensrc/thrift/PlanNodes.thrift

```diff
@@ -150,7 +136,7 @@ struct TBrokerRangeDesc {
     //  it's usefull when format_type == FORMAT_JSON
     14: optional bool num_as_string;
     15: optional bool fuzzy_parse;
-    16: optional THdfsParams hdfs_params
+    16: optional map<string, string> hdfs_params;
```

Comment:
In this way, when there are new and old versions mixed, there will be compatibility problems